lib/capistrano/tasks/maintenance.rake

```diff
@@ -21,16 +21,16 @@ namespace :forkcms do
       # Stop if the maintenance folder exists
       return if capture("if [ -d #{shared_path}/maintenance ]; then echo 'yes'; fi").chomp == 'yes'
 
-      local_maintenance_path = "#{fetch File.dirname(__FILE__)}/../../maintenance"
+      localMaintenancePath = "#{fetch File.dirname(__FILE__)}/../../maintenance"
 
       # Create a maintenance folder containing the index page from our gem
       execute :mkdir, "#{shared_path}/maintenance"
 
       # copy the contents of the index.html file to our shared folder
-      upload! File.open(local_maintenance_path + '/index.html'), "#{shared_path}/maintenance/index.html"
+      upload! File.open(localMaintenancePath + '/index.html'), "#{shared_path}/maintenance/index.html"
```

Comment:
In ruby werken ze met _ ipv camelcase

Reply:
ah ist echt? Dan ga ik dat in mijn volgende PR's ook allemaal moeten veranderen :p zal de commit verwijderen

Reply:
@tijsverkoyen variabelen zijn nu met `_`